src/openrct2/cmdline_sprite.c

```diff
@@ -590,18 +591,35 @@ int cmdline_for_sprite(const char **argv, int argc)
 		sprite_file_close();
 		return 1;
 	} else if (_strcmpi(argv[0], "append") == 0) {
-		if (argc < 3) {
-			fprintf(stderr, "usage: sprite append <spritefile> <input>\n");
+		if (argc != 3 && argc != 5) {
+			fprintf(stderr, "usage: sprite append <spritefile> <input> [x_offset y_offset]\n");
```

Comment:
Maybe write this as `[<x offset> <y offset>]` or `[x offset] [y offset]` instead. But since this is the only one taking more than one optional argument I'm not sure if the rest should be changed as well.